tests/testing_utils.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+def raise_error(message):
+    raise Exception(message)
+
+NACHET_SCHEMA = os.getenv("NACHET_SCHEMA") #or raise_error("NACHET_SCHEMA is not set")
```

Comment:
Doesn't look right that we are 1) ignoring missing environment variables 2) using environment variables in testing instead of mocks

Reply:
To address your second concern, I based this test on the following : https://github.com/ai-cfia/ailab-db/blob/main/tests/testing_utils.py

For the first concern, I ignored the missing env variables since they raised an error even after I've added the secrets into the actions. I will investigate if I can fix this issue.

Reply:
Do you have an update? I see you reinstiated the exception.